src/app.rs

```diff
@@ -221,6 +221,11 @@ pub fn app() -> App<'static, 'static> {
         .arg(Arg::with_name("rust-enum").long("rust-enum").help(
             "Emit a Rust enum and a table that maps codepoints to bidi class.",
         ))
+        .arg(
+            Arg::with_name("combined").long("combined").help(
```

Comment:
It's better if you put this up with the other args and just clone it in. I don't know why --enum and --rust-enum don't, but they also have slightly varying help text for each one, which is probably why.

Reply:
That makes sense. I was using rust-enum as a reference.